socorro/external/postgresql/adi.py

```diff
@@ -51,19 +51,40 @@ def get(self, **kwargs):
                 product_adu
             LEFT OUTER JOIN product_versions pv USING (product_version_id)
             WHERE
-                pv.product_name = %(product)s
-                AND pv.version_string IN %(versions)s
-                AND os_name IN %(platforms)s
-                AND adu_date BETWEEN %(start_date)s AND %(end_date)s
+                pv.product_name = %%s
+                AND (%s)
+                AND os_name IN %%s
+                AND adu_date BETWEEN %%s AND %%s
```

Comment:
Yeah, I've seen code where it creates a "numbered key". You know, that's maybe not such a bad idea. Long tuples are stupid and more dangerous. I think I want to change to use a dict where I do something like:
```python
for i, version in enumerate(version):
    sql_params['version{}'.format(i)] = version
```